Per most recent build failure on f19+:

kdenetwork-4.10.2/kopete/libkopete/avdevice/videodevice.cpp: In member function 'const char* Kopete::AV::VideoDevice::getUnifiedV4L2StdCtrlName(quint32)':
kdenetwork-4.10.2/kopete/libkopete/avdevice/videodevice.cpp:3064:8: error: 'V4L2_CID_HCENTER' was not declared in this scope
   case V4L2_CID_HCENTER:   return I18N_NOOP("Horizontal Center");
        ^
kdenetwork-4.10.2/kopete/libkopete/avdevice/videodevice.cpp:3065:8: error: 'V4L2_CID_VCENTER' was not declared in this scope
   case V4L2_CID_VCENTER:   return I18N_NOOP("Vertical Center");
diff -ur a/kopete/libkopete/avdevice/videodevice.cpp b/kopete/libkopete/avdevice/videodevice.cpp
--- a/kopete/libkopete/avdevice/videodevice.cpp	2013-03-01 07:22:39.148721688 +0200
+++ b/kopete/libkopete/avdevice/videodevice.cpp	2013-04-12 16:39:12.372576267 +0300
@@ -3061,8 +3061,12 @@
 		case V4L2_CID_GAIN:			return I18N_NOOP("Gain");
 		case V4L2_CID_HFLIP:			return I18N_NOOP("Horizontal Flip");
 		case V4L2_CID_VFLIP:			return I18N_NOOP("Vertical Flip");
+#ifdef V4L2_CID_HCENTER
 		case V4L2_CID_HCENTER:			return I18N_NOOP("Horizontal Center");
+#endif
+#ifdef V4L2_CID_VCENTER
 		case V4L2_CID_VCENTER:			return I18N_NOOP("Vertical Center");
+#endif
 #ifdef V4L2_CID_POWER_LINE_FREQUENCY	// since kernel 2.6.25
 		case V4L2_CID_POWER_LINE_FREQUENCY:	return I18N_NOOP("Power Line Frequency");
 		case V4L2_CID_HUE_AUTO:			return I18N_NOOP("Automatic Hue");
